Composer/packages/extensions/plugin-loader/src/composerPluginRegistration.ts

```diff
@@ -121,7 +121,7 @@ export class ComposerPluginRegistration {
       const method = this.loader.webserver[type.toLowerCase()];
 
       if (typeof method === 'function') {
-        method(url, ...handlers);
+        method.bind(this.loader.webserver)(url, ...handlers);
```

Comment:
I think it's better to use `call` in this case.

```suggestion
        method.call(this.loader.webserver, url, ...handlers);
```